pallets/profiles/src/lib.rs

```diff
@@ -25,26 +29,60 @@ pub mod pallet {
 
     #[pallet::pallet]
     #[pallet::generate_store(pub (super) trait Store)]
+    #[pallet::without_storage_info]
     pub struct Pallet<T>(_);
 
     #[pallet::storage]
     #[pallet::getter(fn profile_space_by_account)]
-    pub type ProfileSpaceByAccount<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::AccountId, SpaceId>;
+    pub type ProfileSpaceByAccount<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, SpaceId>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn account_followers)]
+    pub(super) type AccountFollowers<T: Config> =
+        StorageMap<_, Twox64Concat, T::AccountId, Vec<T::AccountId>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn account_followed_by_account)]
+    pub(super) type AccountFollowedByAccount<T: Config> =
+        StorageMap<_, Blake2_128Concat, (T::AccountId, T::AccountId), bool, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn accounts_followed_by_account)]
+    pub(super) type AccountsFollowedByAccount<T: Config> =
+        StorageMap<_, Twox64Concat, T::AccountId, Vec<T::AccountId>, ValueQuery>;
```

Comment:
The names of these storages are somewhat similar and confusing. I know this PR is just to move it as is from solochain, but we need to think of something better in the future.

Reply:
Yeah, definitely. We're going to refactor them. They can be easily merged into 1 storage, which will be `StorageDoubleMap`.